sdk/test-utils/recorder/README.md

```diff
@@ -13,9 +13,9 @@ questions:
 - How to write tests that support parallelism?
 - How to write isomorphic tests for NodeJS and the Browsers?
 
-Our test-utils-recorder attempts to provide an answer for those questions,
-as you'll be able to see throughout this README.
- 
+Our NPM package `@azure/test-utils-recorder` attempts to provide an answer for
```

Comment:
It's not an npm package. It's not published to npm.
Users must follow the conventions of this repo and must use the rush infrastructure to leverage `@azure/test-utils-recorder`.